apps/cli/src/compose/rollups/docker-compose-anvil.yaml

```diff
@@ -10,15 +10,12 @@ services:
             ]
         healthcheck:
             test: ["CMD", "eth_isready"]
-            interval: 10s
-            timeout: 1s
-            retries: 5
+            interval: 200ms
+            timeout: 500ms
+            retries: 20
         environment:
             ANVIL_IP_ADDR: 0.0.0.0
         ports:
             - 8545:8545
-
-    prompt:
-        image: debian:bookworm-slim
-        environment:
-            PROMPT_TXT_01_ANVIL: "Anvil running at http://localhost:8545"
+        volumes:
```

Comment:
why db volume in anvil?

Reply:
because it's the container we have always running that uses the `cartesi/sdk` image which has the initialized database

ugly, I know

Reply:
having another container with a `sleep infinity` looked worse for me

do you have any other idea?

Reply:
you mean, make the cartesi/sdk based on postgres?

or maintain another container image cartesi/postgres ?